specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json

```diff
@@ -1075,6 +1107,39 @@
           }
         }
       },
+      "patch": {
+        "operationId": "ContinuousDataExports_Update",
+        "summary": "Update an existing continuous data export via PATCH.",
+        "x-ms-examples": {
+          "Update export": {
+            "$ref": "./examples/continuousdataexports_update.json"
+          }
+        },
+        "parameters": [
+          {
+            "in": "path",
+            "name": "export_id",
+            "description": "Unique ID for the continuous data export.",
+            "type": "string",
+            "required": true
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Data export patch body.",
+            "schema": {},
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ContinuousDataExport"
+            }
+          }
+        }
+      },
       "delete": {
         "operationId": "ContinuousDataExports_Remove",
```

Comment:
Why `Remove` and not `Delete`?

Reply:
For some generated languages, delete is keyword, for example in typescript, generated code change to "deletemethod()", but use remove we can keep it "remove().